docs/introduction/getting_started.rst

```diff
@@ -128,81 +178,120 @@ the calculation of the NMR spectrum. Lets create the :ref:`simulator_api` object
     sim.run()
 
-The simulated spectrum is calculated and stored in the method object. Next we process and
-plot the data
+The simulated spectrum is stored as a CSDM object in the Method object at
+``sim.methods[0].simulation``. To match an experimental MAS spectrum, however,
+you still need to add some line broadening to the simulated spectrum. For this,
+you can use the :ref:`signal_processor_documentation` object described in the
+next section.
 
-.. note:: In ``mrsimulator``, all resonance frequencies are calculated assuming the
-    weakly-coupled (Zeeman) basis for the spin system.
 
-Signal Processing
------------------
+SignalProcessor
+---------------
 
-``mrsimulator`` performs all calculations in the frequency domain, so plotting the dataset now
-would show only delta functions. For this reason, we have the :ref:`signal_processing_documentation`
-object which applies post-processing to the data after simulation.
+A :ref:`signal_processor_api` object holds a list of operations applied
+sequentially to a dataset. For a comprehensive list of operations and further
+details on using the :ref:`signal_processor_api` object, consult
+the :ref:`signal_processor_documentation` documentation.
+
+Use the code below to create a SignalProcessor object that performs a
+convolution of the simulated spectrum with a Lorentzian distribution having a
+full-width-half-maximum of 200 Hz. This is done with three operations: the
+first operation applies an inverse fast Fourier transform of the spectrum into
+the time domain, the second operation applies a time-domain apodization with an
+exponential decay, and the third operation applies a fast Fourier transform
+back into the frequency domain.
 
-Here we apply 200 Hz of exponential line broadening.
 
 .. plot::
     :context: close-figs
 
-    from mrsimulator import signal_processing as sp
+    from mrsimulator import signal_processor as sp
 
     # Create the SignalProcessor object
     processor = sp.SignalProcessor(
-        operations=[
+        operations = [
             sp.IFFT(),
-            sp.apodization.Exponential(FWHM="200 Hz"),
+            sp.apodization.Exponential(FWHM = "200 Hz"),
             sp.FFT(),
         ]
     )
 
-    # Apply the processor to the simulation data
-    processed_data = processor.apply_operations(data=sim.methods[0].simulation)
-
-Each :ref:`signal_processing_api` object has a list of operations which are applied sequentially to
-a dataset. For a comprehensive list of operations and how to use the signal processing object,
-see the :ref:`signal_processing_documentation` documentation page.
+    # Apply the processor to the simulation dataset
+    processed_simulation = processor.apply_operations(dataset = sim.methods[0].simulation)
 
-Plotting the Data
------------------
 
-We end this example by using the `matplotlib <https://matplotlib.org/stable/>`_ Python library
-to plot the simulated dataset.
+PyPlot
```

Comment:
I think a better section name would be **Visualization with matplotlib**

Also, I've rarely seen the name PyPlot be used when referencing the ``matplotlib.pyplot`` submodule; usually people just use ``matplotlib`` in their language.

"You can use pyplot submodule from matplotlib to visualize simulations and datasets in native Python; see <link> for a brief tutorial on PyPlot"

@pjgrandinetti @deepanshs thoughts?

Reply:
i used the key object names for secrion titles here.  from what i read at the matplotlib, it seems that PyPlot
is the primary object in matplotlib for our users.

Reply:
also, throughout the introduction, the discussion is in terms of 2nd person singular, i.e. "you." Try to avoid writing in terms of "we."